lib/gds_api/test_helpers/publishing_api.rb

```diff
@@ -75,6 +75,29 @@ def stub_publishing_api_isnt_available
         stub_request(:any, /#{PUBLISHING_API_ENDPOINT}\/.*/).to_return(status: 503)
       end
 
+      # Stub a PUT /v2/paths/:base_path request with the given content id and request body.
+      #
+      # @example
+      #   stub_publishing_api_path_reservation(
+      #     my_content_id,
+      #     publishing_app: "content-publisher",
+      #     override_existing: true,
+      #   )
+      #
+      # @param content_id [UUID]
+      # @param body  [String]
+      # @param response_hash [Hash]
+      def stub_publishing_api_path_reservation(base_path, body, response_hash = {})
+        url = PUBLISHING_API_ENDPOINT + "/paths#{base_path}"
+        response = {
+          status: 200,
+          body: "{}",
+          headers: { "Content-Type" => "application/json; charset=utf-8" },
```

Comment:
Any reason we've done this differently to https://github.com/alphagov/gds-api-adapters/blob/ecec6d7cda2d4d4fdce8c35e1e901029f33de329/lib/gds_api/test_helpers/publishing_api.rb#L104 ?

Reply:
Nope. I just copied the response from the publishing-api V2 helpers.